handbook/ops/bizops/user_definitions.md

```diff
@@ -1,37 +1,67 @@
 
 | Metric      | Description                                                      |
 | ----------- | ---------------------------------------------------------------- |
+| Visitor     | Users who accessed the site                                      |
+| Active      | Users who performed a qualifying event (see below)               |
 | Retained    | Users who were active last month _and_ this month                |
 | Churned     | Users who were active last month but not this month              |
 | Resurrected | Users who were _not_ active last month but are active this month |
 | Created     | Users whose account was created this month                       |
 | Deleted     | Users whose account was deleted this month                       |
 
-### Engagement ratios
+### What is a visitor?
+
+A visitor is a unique person who showed up on the site and did anything (or nothing at all, apart from viewing the page.) This is important to measure because each of these users is potentially an active user, but it's also important that we don't count anyone who lands on the site as an active user so that we can measure success of our features that are intended to convert them.
+
+### What is an active user?
+
+An active user is differentiated from a visitor by performing a qualifying event, which are intended to represent someone who has actively **engaged with** and **received value** from the product; something that takes a user from their first Search to establishing a habit around the core value proposition of the app. Qualifying events are:
+
+| Product area              | Qualifying event                   | Description                                                                                         |
+| ------------------------- | ---------------------------------- | --------------------------------------------------------------------------------------------------- |
+| Searching for code        | `SearchSubmitted`                  | Submit a search                                                                                     |
+| Navigating the code graph | `findReferences`, `goToDefinition` | Click on go to definition or fin references. Hovering is excluded as it can be performed passively. |
+| Code Insight              | TBD                                | TBD at launch on cloud                                                                              |
+| Batch Changes             | TBD                                | TBD at launch on cloud                                                                              |
+| API docs                  | TBD                                | TBD                                                                                                 |
+
+We are also looking at the following as possibilities:
+
+- Adding private code
+- TODO identify meaningful events from Reforge
```

Comment:
This was an open discussion item from the source document that we should complete.

Reply:
@anna-mmikhova was the one tagged in the discussion, so if anyone has any ideas of additional items to add then please shout, otherwise I'll remove this section before merging and we can always follow up with additional improvements later.

Reply:
Removing since nobody had any ideas.

Reply:
```suggestion
```